core/src/main/java/nl/nn/adapterframework/jdbc/dbms/GenericDbmsSupport.java

```diff
@@ -387,4 +389,62 @@ public String getBooleanFieldType() {
 	public String getBooleanValue(boolean value) {
 		return (""+value).toUpperCase();
 	}
+
+	@Override
+	public String convertQuery(Connection conn, String query, String dbmsFrom) throws SQLException, JdbcException {
+		if (isQueryConvertable(dbmsFrom)) {
```

Comment:
Deze logica klopt niet. 
Moet zijn: als converteerbaar, dan converteren;
anders zonodig waarschuwing, en ongeconverteerd teruggeven.

Reply:
De logica die jij noemt zit in de verschillende child classesvan GenericDbmsSupport. Wat in GenericDbmsSupport zit is enkel de default voor de child classes welke geen convertQuery hebben.